tools/gmhazard_scripts/db_creation/empirical_db/common.py

```diff
@@ -170,6 +172,8 @@ def write_data_and_close(
             ims=np.asarray(im_list, dtype=str),
         )
         imdb_dict[imdb_keys].close()
+        if rupture_lookup:
+            imdb_dict[imdb_keys].add_rupture_lookup(imdb_dict[imdb_keys].db_ffp, 1)
```

Comment:
As far as I understand this is called by the MPI master process at the end, in which case there is no reason to not set `n_procs` to all available process?

Reply:
Didn't see Jason's comment, so probably ignore what I said above haha.